android/lib/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.29.2'
-  compile "com.google.android.gms:play-services-base:9.2.0"
-  compile 'com.google.android.gms:play-services-maps:9.2.0'
+  compile 'com.facebook.react:react-native:+'
+  compile "com.google.android.gms:play-services-base:8.4.0"
+  compile 'com.google.android.gms:play-services-maps:8.4.0'
```

Comment:
@felipecsl At this point I'm starting to think what you guys are doing different internally that seems to work. As you can see, even after I brought this issue up many times people are still finding it problematic.

As far as play services is concerned. We've had a few concerns with new phones that have an outdated version of play services. In order for the phone the auto update it, the consumer needs to login with their google account. What was the rationale to bumping the versions of play services up? Was there a change that required this?

At this point, I just work off my own fork of the project.